src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -427,45 +435,51 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
         }
     }
 
-    @computed get dataTypeOptions() {
-        let dataTypeIds:string[] = [];
+    @computed get geneToDataTypeOptions():{[entrezGeneId:number]:{value:string, label:string}[]} {
```

Comment:
it's a slipperly slope.  if people see this as example, they will start doing it more in scenarios where it's more inappropriate.

Reply:
I dont see how the memoized method/pure function doesnt do the same? either way we need to funnel the mobxpromise into the @computed

I dont see it as slippery slope, this is a particular case where status doesnt matter to the consumer, and code would be a little more complicated if we put the status checks elsewhere

Reply:
memoized function is different because result is passed in directly (de-referenced).  status is checked outside in view, which is where the component can properly be hidden if its data is pending.  the basic principle we are trying to enforce is, don't show UI when its data dependencies are loading.   making a rule about the above is a good way of enforcing that